src/exchange/shared-account/update/shared-account-update.controller.js

```diff
@@ -120,7 +120,7 @@ angular.module('Module.exchange.controllers').controller(
         this.$routerParams.productId,
       )
         .then((data) => {
-          this.optionsToUpdateAccount = data;
+          this.optionsToUpdateAccount = _.cloneDeep(data);
```

Comment:
Is there any reason to not use [`angular.copy`](https://docs.angularjs.org/api/ng/function/angular.copy)?